modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -155,6 +158,8 @@ const setup = (editor: Editor): RenderInfo => {
     const hasMultipleToolbar = Options.isMultipleToolbars(editor);
     const hasToolbar = Options.isToolbarEnabled(editor);
     const hasMenubar = Options.isMenubarEnabled(editor);
+    const shouldHavePromotion = Options.promotionEnabled(editor) && !Options.hasApiKey(editor);
```

Comment:
This logic would likely be better off in the `Options` module so it's consistent across the board (e.g. `promotionEnabled` should return false if an API key is enabled).

Reply:
Actually, @ephox-mogran reminded me that this doesn't even really make sense to do the API key check within TinyMCE. We'd be better adding the `promotion: false` to Tiny Cloud.